### PR TITLE
-Removed METER_DAMAGE and replaced uses with  METER_CAPACITY. They ar…

### PR DESCRIPTION
**Pull request by geoffthemedio**
…e essentially the same thing, except damage is a capacity for weapons.
-Leads to a couple of non-intuitive uses of  METER_CAPACITY to look up ship damage amounts in the UI, but these can be resolved later.
-Left parsing / Python API support for "Damage" (meters and  effects) which evaluates to the same as "Capacity" equivalents.
-Added METER_MAX_CAPACITY

### CODE REVIEW

**Comment by geoffthemedio** [comment]
Probably best to wait until after v0.4.5, but making a branch for comment before that. Removes the damage meter, having weapon parts use Capacity like other parts. Some quirks as a result in some UI code. Also does some grooming.

File: universe/Effect.cpp [hunk 1]
```diff
@@ -681,13 +681,8 @@ std::string SetShipPartMeter::Description() const {
 std::string SetShipPartMeter::Dump() const {
     std::string retval = DumpIndent();
     switch (m_meter) {
-        case METER_DAMAGE:              retval += "SetDamage";              break;
-        case METER_SPEED:               retval += "SetSpeed";               break;
```
Comment (Cjkjvfnby) [inline universe/Effect.cpp:685]: I don't understand what happens but removing other meters looks strange for me.

**Comment by Cjkjvfnby** [comment]
Big part of this PR is grooming. Does your git tools allow to split them into separate commit and merge to master?

File: universe/Effect.cpp (hunk 1 above)
Comment (geoffthemedio) [inline universe/Effect.cpp:685]: This is more grooming / stale code removal. There were previous plans to have a bunch of part meters to track things like fighter speed, structure, or detection. Presently the only part meter that matters for anything is capacity, so is the only part meter setting effect that needs to be supported.

**Comment by geoffthemedio** [comment]
grooming mixed in with other code changes happens because I see stuff to groom while making other changes.

I can manually undo changes on select lines, commit a partially modified file, then redo the changes, then make another commit, but it's a bunch of effort I'm not inclined to make...

If grooming changes are in separate files, I'll generally commit that separately, as it's no extra work to do so.

**Comment by Cjkjvfnby** [comment]
During work you can add only part of changes to commit. (https://git-scm.com/book/es/v2/Git-Tools-Interactive-Staging) This allow to make separate commits in easy way.

**Comment by geoffthemedio** [comment]
That tool looks overly cumbersome (consistent with how over-complicated git is to do most things, and preference for command line when I'd prefer a GUI, I suppose). 

Were I to do attempt to break up changes within a file into separate commits, I'd usually do so by selectively reverting, saving, committing, undoing, and re-saving within a graphical diff viewer.

But it would not be easy / trivial, and isn't something I'm going to do every time, as I have lots of other things I'd rather spend time on.

**Comment by genestack-solomatin** [comment]
It looks awesome if you use it :)  

Some GUI tools that can do it mentioned in this thread: http://stackoverflow.com/questions/1085162/commit-only-part-of-a-file-in-git/25265642#25265642

**Comment by Dilvish-fo** [comment]
Yes, that's one of the things I like about SmartGit, it has a pretty good side-by-side diff-type tool that you can use for singling out just some of the changed blocks or even single lines within a changed block; I use this sometimes for staging just portions of a changed file for commit, and it's also pretty much the same interface for resolving conflicts.

**Pull request closed without merging**